docusaurus/docs-internals/editor-controls/data-controls/customCss.md

```diff
@@ -0,0 +1,212 @@
+---
+sidebar_position: 1
+toc_max_heading_level: 4
+---
+# Custom Css
```

Comment:
don't use custom css, this is a code mirror, not a textarea only for custom CSS

remove everywhere you used it as "Custom Css"